Timhowie okta 441197 (#2885)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/scim/submit.md

* **Link to Runscope CRUD test results** &mdash; this should be a link to the Runscope site test results showing that your application has successfully passed the CRUD test suite. See [Test your Okta integration](/docs/guides/build-provisioning-integration/test-scim-app/).

>   >**Note:** Runscope CRUD test results are not required if your application only supports the Import feature. Instead, enter a link to your SPEC test results for this field.

Reviewer: ```suggestion
  >**Note:** Runscope CRUD test results aren't required if your application only supports the Import feature. Instead, enter a link to your SPEC test results for this field.
```

Author: Changed as per suggestion.
